src/server.h

```diff
@@ -338,22 +338,22 @@ class CServer :
     OpusCustomDecoder*         OpusDecoderMono[MAX_NUM_CHANNELS];
     OpusCustomEncoder*         OpusEncoderStereo[MAX_NUM_CHANNELS];
     OpusCustomDecoder*         OpusDecoderStereo[MAX_NUM_CHANNELS];
-    CConvBuf<int16_t>          DoubleFrameSizeConvBufIn[MAX_NUM_CHANNELS];
-    CConvBuf<int16_t>          DoubleFrameSizeConvBufOut[MAX_NUM_CHANNELS];
+    CConvBuf<float>            DoubleFrameSizeConvBufIn[MAX_NUM_CHANNELS];
+    CConvBuf<float>            DoubleFrameSizeConvBufOut[MAX_NUM_CHANNELS];
 
     CVector<QString>           vstrChatColors;
     CVector<int>               vecChanIDsCurConChan;
 
-    CVector<CVector<double> >  vecvecdGains;
-    CVector<CVector<double> >  vecvecdPannings;
-    CVector<CVector<int16_t> > vecvecsData;
+    CVector<CVector<float>>    vecvecdGains;
```

Comment:
I put a space in between the >> to make clear it is not the operator>>. Please check the other places, too.

Reply:
Fixed.